### PR TITLE
Function trailing commas

### PR DESCRIPTION
**Pull request by Crell**
* Trailing commas in function parameter list.
* Trailing commas in use clause.
* Minor language change along the way.
* I have no idea what the difference is between para and simplepara...

### CODE REVIEW

File: language/functions.xml [hunk 1]
```diff
@@ -200,7 +200,31 @@ function takes_array($input)
...
+    <title>Passing arrays to functions</title>
+    <programlisting role="php">
+     <![CDATA[
+<?php
+function takes_many_args(
+    $first_arg,
+    $second_arg,
+    $a_very_long_argument_name,
+    $arg_with_default = 5,
+    $again = 'a default string', // This trailing comma is optional as of PHP 8.0.0
```
Comment (Girgias) [inline language/functions.xml:219]: I feel how this comment is written it means that prior to PHP 8.0.0 the trailing comma was mandatory

**Comment by cmb69** [comment]
`<simpara>` is somewhat like a HTML `<p>`, but `<para>` is more like a `<div>`. See https://tdg.docbook.org/tdg/4.5/para.html#d0e133584 for a more specific explanation.

That said, I think that currently PhD treats both pretty similar, and in practise, the documentation mostly uses `<para>`.

**Comment by Girgias** [comment]
> 
> 
> `<simpara>` is somewhat like a HTML `<p>`, but `<para>` is more like a `<div>`. See https://tdg.docbook.org/tdg/4.5/para.html#d0e133584 for a more specific explanation.
> 
> That said, I think that currently PhD treats both pretty similar, and in practise, the documentation mostly uses `<para>`.

I think the only important distinction is when it's inside a `<note>`, `<warning>`, etc. element. With a simpara you'll get something like:
```
Note: simpara content
```
whereas with a para you get:
```
Note:

Para content
```

**Comment by Crell** [comment]
Updated the comment.

**Review by Girgias — APPROVED** [review]
LGTM :)

**Comment by cmb69** [comment]
Thanks!

It seems that the manual is generally sloppy regarding the term "argument", so I left it as is, but in the long run we may consider to distinguish between "parameter" and "argument" (or "formal parameter" and "actual parameter").

**Comment by Crell** [comment]
What even is the difference between an argument and a parameter?  I've generally seen/used them interchangeably.

**Comment by cmb69** [comment]
See https://stackoverflow.com/questions/156767/whats-the-difference-between-an-argument-and-a-parameter

**Comment by Girgias** [comment]
Huh, I've been getting the proper order wrong then apparently, oh well TIL

**Pull request closed without merging**